developer-guides/doc_guidelines.rst

```diff
@@ -3,20 +3,20 @@
 Documentation Guidelines
 ########################
 
-OPEA Project content is written using the `markdown`_ (``.md``) with `MyST extensions`_ and `reStructuredText`_ markup
-language (``.rst``) with `Sphinx extensions`_, and processed
+OPEA Project content is written using the `markdown`_ (``.md``) with `MyST extensions`_
+and `reStructuredText`_ markup language (``.rst``) with `Sphinx extensions`_, and processed
 using `Sphinx`_ to create a formatted stand-alone website.  Developers can
-view this content either in its raw form as ``.md`` and ``.rst`` markup files, or (with
-Sphinx installed) they can build the documentation using the Makefile
-(on Linux systems) to generate the HTML content. The HTML content can then be
-viewed using a web browser. These ``.md`` and ``.rst`` files are maintained in
-the project's GitHub repos and processed to create the
-`OPEA Project documentation`_ website.
+view this content either in its raw form as ``.md`` and ``.rst`` markup files, or
+build the documentation locally following the :ref:`opea_doc_generation` instructions.
+The HTML content can then be viewed using a web browser. These ``.md`` and
+``.rst`` files are maintained in the project's GitHub repos and processed to
+create the `OPEA Project documentation`_ website.
 
 .. note:: While GitHub supports viewing `.md` and `.rst` content with your browser on the
    `github.com` site, markdown and reST extensions are not recognized there, so the
    best viewing experience is through the `OPEA Project documentation`_ github.io
-   website.
+   website. The github.io site also provides navigation and searching that makes
+   it easier for find and read what you're looking for.
```

Comment:
```suggestion
   it easier to find and read what you're looking for.
```

Reply:
You can have a second career as a grammar/spelling checker :)  Thanks.